src/main/java/org/terracotta/context/extended/StatisticsRegistry.java

```diff
@@ -49,7 +48,9 @@
 
 /**
  * @author Ludovic Orban
+ * @deprecated Use {@link org.terracotta.statistics.extended.StatisticRegistry instead}
```

Comment:
It is related to my PR comment: 

"Deprecated the old StatisticsRegistry and moved the one from platform in statistics project (StatisticRegistry). Let me know if we can clean up unused classes"

I'd like to remove unused classes and do some cleanup and move classes at the right location but this will require more changes. I am ok if you are too.

WDYT @lorban

Reply:
If that's our new way, I think we should remove what's unused and do the new way